webapp/src/components/node.tsx

```diff
@@ -86,31 +93,46 @@ const Node = (props: Props) => {
         </Typography>
     );
 
+    const onVideoStarted = () => {
```

Comment:
Does this function get called every time the video is played/resumed? If so, would it be more efficient to check if the video has already been marked as started and only call the REST API when it is not?

Reply:
Added separate ticket for that